pkg/model/hornet/hash.go

```diff
@@ -3,46 +3,103 @@ package hornet
 import (
 	"fmt"
 
+	"github.com/iotaledger/iota.go/consts"
+	"github.com/iotaledger/iota.go/encoding/t5b1"
 	"github.com/iotaledger/iota.go/trinary"
 )
 
 var (
 	// NullHashBytes is the binary hash of the genesis transaction.
-	NullHashBytes = make(Hash, 49)
+	NullHashBytes = make(Hash, hashBytesSize)
```

Comment:
Does go accept that? hashBytesSize is declared in the next line...

Reply:
Yes, the compiler performs a dependency evaluation for initialization:
> Within a package, package-level variable initialization proceeds stepwise, with each step selecting the variable earliest in declaration order which has no dependencies on uninitialized variables. 
https://golang.org/ref/spec#Package_initialization

Should I rather change the order, though?

Reply:
Oh didn't knew that.
Pls change :)